Producer/ProducerManager.php

```diff
@@ -0,0 +1,66 @@
+<?php
+declare(strict_types = 1);
+
+namespace M6Web\Bundle\KafkaBundle\Producer;
+
+use M6Web\Bundle\KafkaBundle\AbstractManager;
+
+/**
+ * Class Producer
+ * @package M6Web\Bundle\KafkaBundle\Producer
+ *
+ * A class to produce messages
+ */
+class ProducerManager extends AbstractManager
+{
+
+    const ORIGIN = 'producer';
+
+    /**
+     * @var \RdKafka\Producer
+     */
+    protected $entity;
+
+    /**
+     * @var \RdKafka\Topic[]
+     */
+    protected $topics = [];
+
+    /**
+     * @param string       $message
+     * @param integer      $partition
+     * @param integer|null $key
+     *
+     * @return void
+     */
+    public function produce(string $message, int $partition = RD_KAFKA_PARTITION_UA, string $key = null)
+    {
+        $this->checkIfEntitySet();
+
+        $this->populateTopicsFromServer();
```

Comment:
We have to get this information (topics present on the cluster) each time we produce or consume to be up-to-date.